nle/tests/test_envs.py

```diff
@@ -49,6 +49,10 @@ def rollout_env(env, max_rollout_len):
     return reward
 
 
+def term_screen(obs):
+    return "\n".join("".join(chr(c) for c in row) for row in obs["terminal_chars"])
```

Comment:
It's so weird that numpy/Python won't let us convert a uint8 array into bytes/strings more easily than that.